nyx/chain.json

```diff
@@ -0,0 +1,186 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "nyx",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://nymtech.net/",
+  "pretty_name": "Nym",
+  "chain_id": "nyx",
+  "bech32_prefix": "n",
```

Comment:
please register this prefix to https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
PR is here, just waiting on it to be merged. https://github.com/satoshilabs/slips/pull/1480

Reply:
update: this is now merged